### PR TITLE
Feature/kendo update 2023

### PR DESCRIPTION
**Pull request by ericarts**
Additional styling issues resolved

### CODE REVIEW

File: FrontEnd/appsettings.Development.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "GCL": {
     "Environment": "development", //can be development, test, or live
-    "SecretsBaseDirectory": "Z:\\AppSettings\\WiserCore\\FrontEnd\\"
+    "SecretsBaseDirectory": "\\\\149.210.132.136\\Juice projects\\AppSettings\\WiserCore\\FrontEnd\\"
```
Comment (MikeVanMourik) [inline FrontEnd/appsettings.Development.json:4]: Don't use static ip, instead of the Z: drive mapping

File: FrontEnd/package.json [hunk 2]
```diff
@@ -8,21 +8,26 @@
...
+    "@progress/kendo-vue-intl": "^3.11.0",
+    "@progress/kendo-vue-treeview": "^3.11.0",
     "axios": "^1.3.4",
     "codemirror": "^5.65.10",
     "codemirror-lint-eslint": "^1.0.1",
     "csslint": "^1.0.5",
     "jshint": "^2.13.6",
     "jszip": "^3.10.1",
     "luxon": "^3.3.0",
+    "monaco-editor": "^0.39.0",
```
Comment (MikeVanMourik) [inline FrontEnd/package.json:30]: it this a intentional dependency and needed for the Kendo update? or is it unrelated?

**Review by MikeVanMourik — CHANGES_REQUESTED** [review]

**Comment by MikeVanMourik** [comment]
whats the status of this? feels like its been stuck for a while? @GilianJuice / @ericarts ?

**Comment by GilianJuice** [comment]
Closing this because it's too out-dated already. Have to redo this sometime with the latest version of Kendo.

**Pull request closed without merging**